notifications/drf_views.py

```diff
@@ -110,3 +113,34 @@ class SubscriptionViewset(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return Subscription.objects.filter(user=self.request.user)
+
+
+class AlertSubscriptionViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
```

Comment:
We might need to delete the subscriptions.

```suggestion
    viewsets.ModelVewset,
```